src/index.js

```diff
@@ -1,18 +1,25 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import './index.css';
 import App from './App';
 import * as serviceWorker from './serviceWorker';
+import { Router } from "react-router-dom";
+import { createBrowserHistory } from 'history';
+import { applyMiddleware, createStore } from 'redux';
+import { rootReducer } from './redux/rootReducer';
+import { message } from './redux/actions';
+import './index.css';
+
+
+const store = createStore(rootReducer, applyMiddleware());
```

Comment:
import {createStore, compose, applyMiddleware} from 'redux'

const composeEnhancers =
  typeof window === 'object' &&
  window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ ?   
    window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__({
    }) : compose;

const store = createStore(
	rootReducer,
	composeEnhancers(
		applyMiddleware()
	)
	)

it would be nice add devtools to out project